docs/reference/images.md

```
@@ -371,20 +371,24 @@ The common end-to-end flow when bumping image refs (for example, moving config f
 
-Each variant produces a 20GB sparse ext4 image. Actual disk usage is much smaller (typically 2–5GB depending on the variant). Each running shed adds another 2–5GB for its own rootfs copy.
+Each variant produces a 20GB sparse ext4 image. Actual disk usage is much smaller (typically 2–5GB depending on the variant). When a shed is created, its rootfs starts out sharing disk extents with the base image via `clonefile` on macOS/APFS or `FICLONE` (ext4/xfs/btrfs reflink) on Linux — the copy is near-instant and adds near-zero physical bytes until the VM begins writing. Writes diverge on a CoW basis from that point, so long-lived sheds grow gradually.
 
-Use `du -sh` to check actual usage:
+If the filesystem doesn't support reflink (e.g. ext4 without the `reflink=1` option, tmpfs), the backend falls back to Linux's `copy_file_range(2)` or finally to a userspace copy. The strategy is logged once per `shed create` on the server, with format:
 
-```bash
-# VZ
-du -sh ~/Library/Application\ Support/shed/vz/*-rootfs.ext4
-du -sh ~/Library/Application\ Support/shed/vz/instances/*
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)